hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md

```diff
@@ -923,6 +923,102 @@ from previous days, and and choosing a combination of
 retry counts and an interval which allow for the clients to cope with
 some throttling, but not to time out other applications.
 
+## Read-After-Overwrite Consistency
+
+S3Guard provides read-after-overwrite consistency through ETags (default) or
```

Comment:
Maybe add a brief note about client vs. server mode here, and what the default is - to provide some context for when client or server side checks are mentioned later on.

Reply:
Updated